pkg/helmreconciler/listeners.go

```diff
@@ -258,25 +258,41 @@ func (l *LoggingRenderingListener) BeginResource(obj runtime.Object) (runtime.Ob
 
 // ResourceCreated logs the event
 func (l *LoggingRenderingListener) ResourceCreated(created runtime.Object) error {
-	log.Info("new resource created")
+	accessor := meta.NewAccessor()
+	kind, _ := accessor.Kind(created)
+	name, _ := accessor.Name(created)
+	version, _ := accessor.APIVersion(created)
+	log.Infof("resource created (%s, %s, %s)", version, kind, name)
```

Comment:
You can get the full group/version/kind from the runtime object directly, i.e. ```created.GetObjectKind().GroupVersionKind()```

Reply:
Thanks, done.